spec/dap/filter/gquic_filter_spec.rb

```diff
@@ -5,20 +5,35 @@
 
     let(:filter) { described_class.new(['data']) }
 
-    context 'testing gquic valid input' do
+    context 'testing gquic valid input base64 encoded output from the real world' do
       let(:decode) { filter.decode(Base64.decode64("DQAAAAECAwQFUTA0NFEwNDNRMDM5UTAzNQ=="))}
       it 'returns an hash w/ versions as list of versions' do
         expect(decode).to eq({"versions"=> ["Q044","Q043","Q039","Q035"]})
       end
     end
 
+    context 'testing gquic valid input artifical example' do
```

Comment:
hmm.  Should you also now have a test that ensures that you can properly pull out the valid versions even when there are some invalid ones in the same response?